### DISCUSSION TITLE
Can the SQLSysClrTypes and ReportViewer control be used for SQL Server 2019 or 2022?

### DISCUSSION THREAD

**Discussion by 0x7FFFFFFFFFFFFFFF** [microsoft/SqlNexus#184] [Q&A] [answered]
While I was installing `SQLSysClrTypes` and `ReportViewer control`, I noticed they both are SQL Server 2016 version. I downloaded them from [the prerequisite page](https://github.com/microsoft/SqlNexus/releases/tag/7.22.03.23). My SQL Server is 2019 and 2022. Can I still install the 2016 version of  `SQLSysClrTypes` and `ReportViewer control`?

**Comment by PiJoCoder** [comment, marked as answer]
> While I was installing `SQLSysClrTypes` and `ReportViewer control`, I noticed they both are SQL Server 2016 version. I downloaded them from [the prerequisite page](https://github.com/microsoft/SqlNexus/releases/tag/7.22.03.23). My SQL Server is 2019 and 2022. Can I still install the 2016 version of `SQLSysClrTypes` and `ReportViewer control`?

Yes, you can happily use these on a system with SQL 2022 and SQL 2019 - they are client-related DLLs. I have them on my system where I have 2016, 2017, 2019 without any problems.

**Reply by 0x7FFFFFFFFFFFFFFF** [reply]
Thanks for the answer!
